packages/mobile/src/firebase/firebase.ts

```diff
@@ -138,14 +138,13 @@ export function* initializeCloudMessaging(app: ReactNativeFirebase.Module, addre
   })
   yield spawn(watchFirebaseNotificationChannel, channelOnNotification)
 
-  const initialNotification = yield call([app.messaging(), 'getInitialNotification'])
+  // Manual type checking because yield calls can't infer return type yet :'(
+  const initialNotification: Awaited<ReturnType<
+    FirebaseMessagingTypes.Module['getInitialNotification']
+  >> = yield call([app.messaging(), 'getInitialNotification'])
   if (initialNotification) {
-    Logger.info(TAG, 'App opened fresh via a notification')
-    yield call(
-      handleNotification,
-      initialNotification.notification,
```

Comment:
We're were bit again by `redux-saga` types not being inferred correctly. 
Here leading to the initial notification to not be handled correctly.